src/pages/LoginPage/LoginPage.tsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import LoginForm from '../../components/LoginForm';
 import Background from '../../assets/wallpaper.png';
 import { useNavigate } from 'react-router-dom';
+import useAuth from '../../contexts/useAuth';
 import './loginPage.css';
 
 const LoginPage = () => {
   const navigate = useNavigate();
-  const loginRequest = () => navigate('home');
+
+  const { isAuthenticated } = useAuth();
+
+  useEffect(() => {
+    if (isAuthenticated) {
+      navigate('home');
+    }
+  }, [isAuthenticated]);
```

Comment:
I think you are missing the `navigate` method as a dependency for the useEffect hook